### PR TITLE
Kemono: add mirror setting and Coomer new design

### LINKED ISSUE #16866: Kemono: Add support for mirrors
### Source name

Kemono v1.4.5

### Source language

all

### Describe your suggested feature

Kemono announced additional mirror "https://kemono.su/". Please add support for this url to the relevant extension

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #16901: Coomer: No results found
### Source information

Coomer 1.4.5

### Source language

Multi

### Steps to reproduce

1. Press "Browse"
2. In the "Sources" tab press on the "Coomer" extension
3. Either pressing on the "Popular" tab or the "Latest" tab shows "No results found"

### Expected behavior

Usually in this two sections I found the latest and the popular entries

### Actual behavior

It shows "No results found"

### Tachiyomi version

Stable 0.14.6

### Android version

Android 13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by stevenyomi**
Closes #16866
Closes #16901

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/Kemono.kt [hunk 1]
```diff
@@ -50,43 +47,21 @@ open class Kemono(
         Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
     }
 
-    override fun popularMangaRequest(page: Int): Request =
```
Comment (AntsyLich) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/Kemono.kt:53]: Why is this even a HttpSource at this point

Comment (stevenyomi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/Kemono.kt:53]: Then what should it be?

Comment (AntsyLich) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/Kemono.kt:53]: Ignore this dumb shit

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/KemonoGenerator.kt [hunk 2]
```diff
@@ -9,10 +9,10 @@ class KemonoGenerator : ThemeSourceGenerator {
 
     override val themePkg = "kemono"
 
-    override val baseVersionCode = 5
+    override val baseVersionCode = 6
 
     override val sources = listOf(
-        SingleLang("Kemono", "https://kemono.party", "all", isNsfw = true),
+        SingleLang("Kemono", "https://kemono.su", "all", isNsfw = true),
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/KemonoGenerator.kt:15]: The `.su` domains (also available for Coomer) are mirrors, the `.party` are still the main ones, so better make it as a setting instead?

Comment (stevenyomi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/KemonoGenerator.kt:15]: I don't really bother adding a setting. `kemono.party` is blocked here but `coomer.party` isn't, so `.su` for Coomer is less of a problem. In fact, `img.kemono.su` is working but `img.coomer.su` is not, so `.su` works better with Kemono. Attachment server domains at either `.su` aren't working.

Comment (alessandrojean) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/KemonoGenerator.kt:15]: I see, wasn't aware of that. Keep it default to `.su` then but maybe add a setting as the inverse might occur eventually as well, would be better for the users.

Comment (stevenyomi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/kemono/KemonoGenerator.kt]: Added setting.

**Pull request merged**